### ISSUE TITLE
[Feature Request] Main dashboard  with all pages and their status

### ISSUE THREAD

**Issue by trickyut** [msgbyte/tianji#74]
Hello,
Is it possible to create main dashboard  with all configured pages and their current status? If any of the monitor checks configured on a particular page is different than green it should be visible on that dashboard as well. 

Thanks to that it would be quite easy to monitor all pages all at once and react instead of clicking on each page separately and checking if everything is OK.
I used current pages overview just to show my idea 
![tianji_dashboard](https://github.com/msgbyte/tianji/assets/34136159/413f0b41-646d-42d6-abc9-44d3ddef3a85)

**Comment by moonrailgun** [comment]
i am not sure i understand your mind. did you wanna display a overview status in page list?

**Comment by trickyut** [comment]
Yes, i've got ~ 20 applications that i need to monitor. For each application i want to create separate page with 4-5 monitoring checks. I would like to use page list as a main monitoring page where i can easily check if all my apps are working fine or not. If not then i can click on particular page with yellow/red status to check what's wrong.
